docs/source/examples/Using Interact.ipynb

```diff
@@ -375,8 +375,9 @@
     "  <tr><td>`'Hi there'`</td><td>Text</td></tr>\n",
     "  <tr><td>`value` or `(min,max)` or `(min,max,step)` if integers are passed</td><td>IntSlider</td></tr>\n",
     "  <tr><td>`value` or `(min,max)` or `(min,max,step)` if floats are passed</td><td>FloatSlider</td></tr>\n",
-    "  <tr><td>`('orange','apple')` or `{'one':1,'two':2}`</td><td>Dropdown</td></tr>\n",
-    "</table>"
+    "  <tr><td>`['orange','apple']` or `{'one':1,'two':2}`</td><td>Dropdown</td></tr>\n",
+    "</table>\n",
+    "Note that a dropdown is used if a list or a dict is given (signifying discrete choices), and a slider is used if a tuple is given (signifying a range).
```

Comment:
This line is missing a closing quote, throwing a json syntax error when trying to open this